README.md

```diff
@@ -1,2 +1,95 @@
-# github
-Set of semantic-release plugins for publishing a Github release
+# @semantic-release/github
+
+Set of [semantic-release](https://github.com/semantic-release/semantic-release) plugins for publishing a [Github release](https://help.github.com/articles/about-releases).
+
+[![Travis](https://img.shields.io/travis/semantic-release/github.svg)](https://travis-ci.org/semantic-release/github)
+[![Codecov](https://img.shields.io/codecov/c/github/semantic-release/github.svg)](https://codecov.io/gh/semantic-release/github)
+[![Greenkeeper badge](https://badges.greenkeeper.io/semantic-release/github.svg)](https://greenkeeper.io/)
+
+## verifyConditions
+
+Verify the presence and the validity of the `githubToken` (set via option or environment variable).
+
+### Options
+
+| Option                | Description                                 | Default                                                |
+| --------------------- | ------------------------------------------- | ------------------------------------------------------ |
+| `githubToken`         | The token used to authenticate with GitHub. | `process.env.GH_TOKEN` or `process.env.GITHUB_TOKEN`   |
+| `githubUrl`           | The GitHub Enterprise endpoint. Optional.   | `process.env.GH_URL` or `process.env.GITHUB_URL`       |
+| `githubApiPathPrefix` | The GitHub Enterprise API prefix. Optional. | `process.env.GH_PREFIX` or `process.env.GITHUB_PREFIX` |
```

Comment:
Just a thought: instead of `Optional.` I would add `**Required.**` to the beginning of descriptions for required options. What do you think?

Reply:
Done !